# HG changeset patch
# User Rik <rik@octave.org>
# Date 1459458733 25200
#      Thu Mar 31 14:12:13 2016 -0700
# Node ID feac06371be1871391a0de26d2d9b6a075be2e91
# Parent  faa23d2161f8ea5e6f98c40fe468e196ee83e673
unpack.m: Use more specific regexp to prevent accidental hits.

* unpack.m (__parse_gzip__): Make the regexp decoding the output of gzip much more specific so that it doesn't trigger on the name "dir with spaces".

diff --git a/scripts/miscellaneous/unpack.m b/scripts/miscellaneous/unpack.m
--- a/scripts/miscellaneous/unpack.m
+++ b/scripts/miscellaneous/unpack.m
@@ -287,17 +287,17 @@ endfunction
 function output = __parse_tar__ (output)
   ## This is a no-op, but it makes things simpler for other cases.
 endfunction
 
 function files = __parse_gzip__ (output)
   ## Parse the output from gzip and gunzip returning the files
   ## commpressed (or decompressed).
 
-  files = regexprep (output, '^.+ with (.*)$', '$1');
+  files = regexprep (output, '^.+ -- replaced with (.*)$', '$1');
 endfunction
 
 function files = __parse_bzip2__ (output)
   ## Parse the output from bzip2 and bunzip2 returning the files
   ## commpressed (or decompressed).
 
   ## Strip leading blanks and .bz2 extension from filename
   files = regexprep (output, '^\s+(.*)\.bz2: .*', '$1');
